### PR TITLE
Added a check to only save landscape images (Requires PIL)

### PR DESCRIPTION
**Pull request by mahad-ahmed**

### CODE REVIEW

File: extract_lockscreen_images.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import shutil
 from pathlib import Path
 from datetime import datetime
+from PIL import Image
+from PIL.ExifTags import TAGS
```
Comment (AdeelH) [inline extract_lockscreen_images.py]: It doesn't look like `TAGS` is being used anywhere.

File: extract_lockscreen_images.py [hunk 2]
```diff
@@ -11,7 +13,7 @@
 src_dir = Path(f'C:/Users/{username}/AppData/Local/Packages/Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy/LocalState/Assets') # yapf: disable
 curr_date = datetime.today()
 dst_dir = Path(
-    f'C:/Users/{username}/Pictures/lockscreen_images/{curr_date:%Y_%m_%d}')
+    f'C:/Users/{username}/Pictures/lockscreen_images/')
```
Comment (AdeelH) [inline extract_lockscreen_images.py]: This change should be excluded from this PR.

**Review by AdeelH — CHANGES_REQUESTED** [review]
Thanks for the PR! This good to go after a couple of trivial changes.

File: extract_lockscreen_images.py (hunk 1 above)
Comment (mahad-ahmed) [reply extract_lockscreen_images.py]: Forgot to remove that MB.

**Review by AdeelH — APPROVED** [review]

**Pull request merged**